src/amo/components/Footer/index.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import LanguagePicker from 'amo/components/LanguagePicker';
+import translate from 'core/i18n/translate';
+import Icon from 'ui/components/Icon';
+
+import './styles.scss';
+
+
+export class FooterBase extends React.Component {
+  static propTypes = {
+    handleViewDesktop: PropTypes.func.isRequired,
+    i18n: PropTypes.object.isRequired,
+    lang: PropTypes.string.isRequired,
+    location: PropTypes.object.isRequired,
+  }
+
+  render() {
+    const { handleViewDesktop, i18n, lang, location } = this.props;
+    const homepageText = i18n.gettext("Go to Mozilla's homepage");
+
+    return (
+      <footer className="Footer">
+        <div className="Footer-language-and-links">
+          <LanguagePicker location={location}
+            ref={(ref) => { this.languagePicker = ref; }} />
+          <ul className="Footer-links">
+            <li>
+              <a href={`https://www.mozilla.org/${lang}/privacy/websites/`}
```

Comment:
There seem to be certain locales it will redirect to if it doesn't have one available, but they need to be known (so `en-CA` redirects to `en-US`). I'll see about all of them, but given it will 404 if it doesn't redirect, we could just omit the locale to be safe. moz.org should figure out the user's locale like we do so that'll only an annoying experience if you manually set your locale on AMO to be different than the one assigned to you.